browser/src/control/jsdialog/Widget.Combobox.js

```diff
@@ -84,6 +104,28 @@ JSDialog.comboboxEntry = function (parentContainer, data, builder) {
 	return false;
 };
 
+ // Helper function to find the next focusable element using JSDialog.IsFocusable
+ function getNextFocusableEntry(data) {
+	let nextElement;
+	let pos = parseInt(data.pos)
+	do {
+		pos++; // Move to the next position
+		nextElement = document.getElementById(data.comboboxId + '-entry-' + pos);
```

Comment:
please try to avoid using `getElementById`, it's fragile on future changes.
Can we maybe use `JSDialog.GetFocusableElements` which we will just iterate?

or can we create helper: `JSDialog.GetNextFocusableElement` which will find element by iterating the DOM?

Reply:
or maybe even we can just use `JSDialog.KeyboardGridNavigation` ?

Reply:
we shouldn't implement the same ting few times, ideally it should be simple algorithm which uses "siblings" and not any "ids" and just looks for next one focusable on arrow right, previous on arrow left etc...
this way we can use it in any place without modification

Reply:
Yes i can try on that but still i think to get Dropdown container i need to user byID one time.
`const comboboxEntries = document.getElementById(data.id +'-entries');`

Reply:
> EDIT: it seems something exists already:
JSDialog.FindFocusableElement(
  potentialCurrentElement.previousElementSibling,
  'previous'
  );
  
  This will not work as it also return the element if current is focusable we need to pass potential next element to this function and this will work...

Reply:
So indirectly need to get next element and pass to this function. Or better we just create new helper for it ?

Reply:
like this: 
![image](https://github.com/user-attachments/assets/ebc6ed27-285e-48b9-ba23-b0b39bd765a2)

Reply:
> there is helper for that too: `var dropdownRoot = JSDialog.GetDropdown(data.id);`
> 
> we at least need to encapsulate all these "get by id" so it will be single place to modify in the future (if id generation will be changed)

I see okay then it will work

Reply:
Let me do the changes and test, Thanks :)

Reply:
less duplicated code == better :)

I would try:
1.  `JSDialog.KeyboardGridNavigation` (it works left, right, up, down, might be not good)
2. if above doesn't work, please try to create generic: `JSDialog.KeyboardListNavigation` (up,down only)
3. do custom thing for this one